src/storyblok-auth-api/handle-requests/openidClient.ts

```diff
@@ -26,11 +26,19 @@ export const openidClient: CreateOpenIdClient = (params, region) => {
         ? `${getRegionUrl(region)}/oauth/user_info`
         : undefined,
   })
-  return new Client({
+
+  const client = new Client({
```

Comment:
i am wondering if the http options can be set when creating the client ? https://www.npmjs.com/package/openid-client/v/2.4.3#changing-http-request-defaults